cmd/controller/state/imagescan/scanner.go

```diff
@@ -89,7 +89,19 @@ func (s *Scanner) ScanImage(ctx context.Context, params ScanImageParams) (rerr e
 	}
 
 	jobName := genJobName(params.ImageName)
-	vols := volumesAndMounts{}
+	vols := volumesAndMounts{
```

Comment:
Please check how do we pass our local kvisor e2e tests. We somehow get image metadata requests.

Reply:
It's based on the layer size: https://github.com/aquasecurity/trivy/blob/main/pkg/fanal/walker/cached_file.go#L40

Reply:
Which is 200Mb: https://github.com/aquasecurity/trivy/blob/main/pkg/fanal/walker/walk.go#L9